openapi.yaml

```
@@ -2150,11 +2149,8 @@ paths:
                         scopes:
                           type: array
                           description: >-
-                            A list of OpenID Scopes that the client MUST use.
-
-
-                            If empty, the default scopes of OpenID Connect MUST
-                            be used.
+                            A list of OpenID Connect scopes that the client MUST use.
+                            The list MUST at least contain the `openid` scope.
```

```suggestion


                            If no scope is specified, clients MUST use the OpenID Connect
                            default scope `openid`.
```

Please also remove the requirement for scopes here: https://github.com/Open-EO/openeo-api/pull/288/files#diff-fe030a7c1568b3decf599edf399be7f4L2132